### ISSUE TITLE
Batchnorm momentum

### ISSUE THREAD

**Issue by rederyang** [titu1994/Wide-Residual-Networks#21]
According to the definition of momentum in keras doc, the momentum of BN layers should be 0.9, which is not the default momentum value of BN layer of pytorch (0.1).
